ddtrace/internal/remoteconfig/client.py

```diff
@@ -191,7 +191,7 @@ def __init__(self):
         self._client_tracer = dict(
             runtime_id=runtime.get_runtime_id(),
             language="python",
-            tracer_version=ddtrace.__version__,
+            tracer_version=ddtrace.__version__.replace("rc", "-rc", 1),
```

Comment:
I think we are going to also have issues with the `+g<git short hash>` stuff as well.

I really hate to suggest it, but I cannot think of anything better right now... I wonder if we should just manually create the semver from a `package.version.Version`?

I'd suggest using `ddtrace._version.version_tuple`, but we might end up in a case where `ddtrace._version` hasn't been generated and instead we grab the version from the distribution.

It mostly seems we just need to drop the "local" part from the version, and do `<m>.<m>.<m>[-rc<n>]`... but I wish there was a cleaner way, feels so manual.

Reply:
ignore me, I missed the Slack thread where this was already discussed... `'1.5.0-rc2.dev88+gc94cc14b3.d20221027'` would be a semver compliant version, so my concerns can be ignored.